addons/sales_team/models/crm_team.py

```diff
@@ -63,6 +67,29 @@ def _get_default_favorite_user_ids(self):
     dashboard_button_name = fields.Char(string="Dashboard Button", compute='_compute_dashboard_button_name')
     dashboard_graph_data = fields.Text(compute='_compute_dashboard_graph')
 
+    @api.depends('crm_team_member_ids.active')
+    def _compute_member_ids(self):
+        for team in self:
+            team.member_ids = team.mapped('crm_team_member_ids.user_id').ids
```

Comment:
Since Odoo 13.0, `mapped` is built-in (and blazing fast) on relational fields, and `.ids` is not necessary and actually incorrect (prepare yourself for fucked up onchanges)
```suggestion
            team.member_ids = team.crm_team_member_ids.user_id
```